src/main/java/com/greenfox/dramacsoport/petclinicbackend/controllers/PetController.java

```diff
@@ -24,4 +28,9 @@ public ResponseEntity<?> getPets(Principal user) {
         return new ResponseEntity<>(petService.getUserPets(user.getName()), HttpStatus.OK);
     }
 
+    @PostMapping("/api/v1/user/pet/add")
```

Comment:
I think the "add" should be omitted, you wanted to `POST` a new resource which is related to the user and the 'addition' should be expressed by the http method rather than a suffix. 

```suggestion
    @PostMapping("/api/v1/user/pet/")
```

Reply:
corrected